writing-test-plans/test-plan-manifest.md

```diff
@@ -55,6 +125,13 @@ Finally, we have [test cases](../concepts-and-architecture/test-structure.md#tes
 [[testcases]]
 name= "quickstart"
 instances = { min = 1, max = 5, default = 1 }
+
+[testcases.params]
+  conn_count       = { type = "int", desc = "number of TCP sockets to open" default = 5 }
+  conn_outgoing    = { type = "int", desc = "number of outgoing TCP dials", default = 5 }
+  conn_delay_ms       = { type = "int", desc = "random milliseconds jitter before TCP dial", default = 30000 }
+  concurrent_dials = { type = "int", desc = "max number of concurrent net.Dial calls", default = 10 }
+  data_size_kb     = { type = "int", desc = "size of data to write to each TCP connection", default = 128 }
```

Comment:
Are there other `type` options possible? Can we create an option without a default?
Could you replace these very similar examples with one with different types, default, etc? This would give a better view of the options.

Reply:
Testcase will fail if we do not provide default value. I was using these manifests for testing: https://github.com/testground/learning-example-tg/blob/master/manifest.toml . For example if we do not provide 
 `messages   = { type = "int", default = 5}` 
 we get the error: 
       ```
FAIL << single[001] >> strconv.Atoi: parsing "null": invalid syntax
Sep 23 08:37:43.965515	INFO	0.6839s    MESSAGE << single[000] >> claimed sequence numbers; global=1, group(single)=1
Sep 23 08:37:43.965603	INFO	0.6840s       FAIL << single[000] >> strconv.Atoi: parsing "null": invalid syntax
```
I will add that note to the docs